CMakeLists.txt

```diff
@@ -187,8 +187,23 @@ install(
 # Install targets
 #
 
-install(EXPORT uvwConfig NAMESPACE uvw:: DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/uvw)
-install(TARGETS uvw EXPORT uvwConfig ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+set_target_properties(
+    uvw PROPERTIES
+    VERSION ${UVW_VERSION_MAJOR}.0.0
```

Comment:
Why not `UVW_VERSION_MINOR` and `UVW_VERSION_PATCH`? What am I missing?

Reply:
I've just stayed true with libuv [style](https://github.com/libuv/libuv/blob/v1.x/CMakeLists.txt#L725): 
![image](https://user-images.githubusercontent.com/6248626/211108632-d12453bd-1ed3-4166-8f11-74c4bf959ca0.png)
That's clear we can add more details with minor and patch, do you think is better to do so?

Reply:
It's a sort of common practice with autotools, within MAJOR' versions no APIs are broken.. and so on..
but you know what, let's add all info we already have for free in the version property.
I'll add my commit asap.